src/components/InventoryPage/InventoryPage.jsx

```diff
@@ -15,11 +15,13 @@ import './InventoryPage.module.css';
 
 import DonationModal from './DonationModal';
 import { getDonationsFromDB, makeDate } from '../../utils/InventoryUtils';
-
+import PaginationFooter from '../PaginationFooter/PaginationFooter';
+import { PNPBackend } from '../../utils/utils';
```

Comment:
🚫 **[eslint]** <[import/newline-after-import](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/newline-after-import.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after import statement not followed by another import.
```suggestion
import { PNPBackend } from '../../utils/utils';

```